batch/sql/insert_attempt_resources_format_version_lt_3.py

```diff
@@ -70,7 +70,9 @@ async def _process(tx):
                               end_batch_id, end_batch_id, end_job_id, end_batch_id, end_job_id, end_attempt_id)
 
             if size is not None:
-                limit = f'LIMIT {size}'
+                # No risk in this file because all calls use int sizes,
+                # but this is good practice if we copy/paste this in the future:
+                limit = f'LIMIT {int(size)}'
```

Comment:
I think this _theoretically_ could be parametrized. From [docs](https://dev.mysql.com/doc/refman/8.0/en/select.html):
> * Within prepared statements, `LIMIT` parameters can be specified using `?` placeholder markers.
This however is still safe.